x-pack/plugins/apm/public/components/app/Settings/index.tsx

```diff
@@ -44,26 +31,8 @@ export function Settings({ children, location }: SettingsProps) {
 
   return (
     <>
```

Comment:
I don't think this is necessary anymore since <EuiPage/> is wrapping all others components.